lib/rspec/matchers/built_in/be.rb

```diff
@@ -75,7 +75,7 @@ module BeHelpers
       private
 
         def args_to_s
-          @args.empty? ? "" : parenthesize(inspected_args.join(', '))
+          @args.empty? ? '' : parenthesize(inspected_args.join(', '))
```

Comment:
Our own rubocop rules don't enforce this, so in future it'd be good to avoid the git churn.